sdk/compute/Azure.ResourceManager.Compute/tests/Scenario/ProximityPlacementGroupContainerTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Compute.Tests.Helpers;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Compute.Tests
+{
+    public class ProximityPlacementGroupContainerTests : ComputeTestBase
+    {
+        private ResourceGroup resourceGroup;
+        public ProximityPlacementGroupContainerTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+        private ProximityPlacementGroupData BasicProximityPlacementGroupData
+        {
+            get
+            {
+                return ResourceDataHelper.GetBasicProximityPlacementGroupData(DefaultLocation);
+            }
+        }
```

Comment:
We can remove this property and use `ResourceDataHelper.GetBasicProximityPlacementGroupData(DefaultLocation)` wherever is invoking this property